docs/cartesian-product.adoc

```diff
@@ -27,23 +27,21 @@ NOTE: The CartesianTest extension has undergone significant changes in 1.6.0.
 This included moving it into a new package.
 link:/docs/cartesian-product-v1[The old variant] is deprecated and will be removed in the 2.0 release (tentatively scheduled for 2022).
 
-== Basic Use
+== Basic use
 
 `@CartesianTest` is used _instead_ of `@Test` or other such annotations (e.g. `@RepeatedTest`).
 
 You can supply test parameters to `@CartesianTest` in two ways:
 
- - You can annotate your test method, providing all parameter values in a single annotation.
- - The test parameters can be annotated with `@CartesianTest.Values`, `@CartesianTest.Enum`, or range source annotations (see <<Annotating your test parameters>>)
+ - The test parameters can be annotated directly with `@CartesianTest.Values`, `@CartesianTest.Enum`, or range source annotations (see <<Defining arguments on parameters>>)
```

Comment:
I thought we wanted to use active voice, wherever possible?

Reply:
Good point! Fixed in d3cb494.